contributors/devel/README.md

```diff
@@ -100,6 +100,9 @@ See the [kubernetes/release](https://github.com/kubernetes/release) repository f
 * **Structured Logging migration instructions** [migration-to-structured-logging.md](sig-instrumentation/migration-to-structured-logging.md)
 
 ### SIG Storage
+* **CSI Drivers Doc** [CSI drivers doc](https://kubernetes-csi.github.io/docs/)
+  This site documents how to develop, deploy, and test a [Container Storage Interface](https://github.com/container-storage-interface/spec/blob/master/spec.md) (CSI) driver on Kubernetes.
+
 * **Flexvolume** [flexvolume.md](sig-storage/flexvolume.md)
```

Comment:
@msau42 do we still need to keep this doc about Flexvolume here?

Reply:
I think we still should keep it but Add a note here and in that file that csi is the recommended method.

Separately, I wonder if we should deprecate flexvolumes now that csi is mature.

Reply:
it appears to be marked deprecated here: https://github.com/kubernetes/community/blob/master/sig-storage/volume-plugin-faq.md

Reply:
I see.  Sure, I'll add a note about it.

Reply:
I see that it also marked in-tree plugins as deprecated.  I wonder if these general deprecation messages were also added to the release notes?  We added deprecation messages for individual cloud provider volume plugins when their CSI migration moved to beta.  How is that different from these general deprecation messages about in-tree plugins?

```
There are three methods to implement a volume plugin:

In-tree volume plugin [deprecated]
Out-of-tree FlexVolume driver [deprecated]
Out-of-tree CSI driver
```

Reply:
Added note about deprecation of flexvolume.